tests/e2e/e2e.mk

```diff
@@ -38,3 +38,24 @@ test-e2e-aws-simple-1-20: test-e2e-install
 		--test-package-marker=stable-1.20.txt \
 		--parallel 25 \
 		--skip-regex="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|RuntimeClass|RuntimeHandler"
+
+.PHONY: test-e2e-do-simple-1-20
+test-e2e-do-simple-1-20: test-e2e-install
+	kubetest2 kops \
+		-v 6 \
+		--up --down \
+		--cloud-provider=digitalocean \
+		--cluster-name e2e-test-do.k8s.local \
+    --create-args "--networking=cilium --api-loadbalancer-type=public --master-count=3" \
+		--env S3_ENDPOINT=sfo3.digitaloceanspaces.com \
+		--env KOPS_STATE_STORE=do://e2e-kops-space \
+		--env JOB_NAME=pull-kops-e2e-kubernetes-do-kubetest2 \
+		--kops-version-marker=https://storage.googleapis.com/kops-ci/bin/latest-ci-updown-green.txt \
+		--kubernetes-version=https://storage.googleapis.com/kubernetes-release/release/stable-1.20.txt \
+		--template-path=tests/e2e/templates/simple.yaml.tmpl \
```

Comment:
```suggestion
```

lets leave this flag out since it affects the resulting cluster manifest and i'm not sure what might need to change for that template to work with DO. without the flag it will use `kops create cluster --cloud digitalocean` like normal.

Also I think we should set the state store here rather than an `--env` flag:

https://github.com/kubernetes/kops/blob/cb1edd047eaa683f1411a6dd600ee2c4cd7bf6a8/tests/e2e/kubetest2-kops/deployer/common.go#L186-L197